core/templates/components/button-directives/social-buttons.component.html

```diff
@@ -26,6 +26,12 @@
         <span class="oppia-icon-accessibility-label">Twitter</span>
       </a>
     </div>
+    <div class="oppia-github-follow">
+      <a href="https://github.com/oppia" target="_blank" rel="noopener">
+        <i class="oppia-follow-icons fa-brands fa-github github-icon"></i>
```

Comment:
Let's use `oppia-follow-github-icon` name to avoid any class name conflict.

Reply:
Hi @DubeySandeep PTAL,I have changed the name to oppia-follow-github-icon.
Please check the commit named:**Changes after review.**